CMakeLists.txt

```diff
@@ -60,24 +60,18 @@ file(GLOB_RECURSE SRC_SILO "src/*.cpp")
 list(REMOVE_ITEM SRC_SILO ${SRC_TEST})
 
 # ---------------------------------------------------------------------------
-# Linter
+# Targets
 # ---------------------------------------------------------------------------
 
-option(BUILD_WITH_CLANG_TIDY "Build process clang-tidy")
```

Comment:
Can you also change the `build_with_conan.py` file to stop using this variable, but using `-D CMAKE_CXX_CLANG_TIDY=clang-tidy`? 
I tried `cmake_options.append("-D CMAKE_CXX_CLANG_TIDY=clang-tidy")` myself, but I get a lot of errors. Some dont seem to find std packages...
```
/fasta_format_exception.h:3:10: error: 'stdexcept' file not found [clang-diagnostic-error]
    3 | #include <stdexcept>
```

Reply:
This should be resolved as I took another approach which left all relevant files for manual linting unchanged